assets/css/app.css

```diff
@@ -371,6 +371,14 @@ ul.sidebar .sidebar-list .sidebar-sublist a.active {
   float: none !important;
 }
 
+.bootbox-form .bootbox-input-checkbox {
```

Comment:
In order to workaround that loop through results thing, would that be possible to transform the input-checkbox from bootbox using the `switch` css rules?